spec/graphql/schema_spec.rb

```diff
@@ -11,11 +11,23 @@ class DummyFeature2 < GraphQL::Schema::Directive::Feature
 
     end
 
+    class Query < GraphQL::Schema::Object
+      field :some_field, String, null: true
+    end
+
+    class Mutation < GraphQL::Schema::Object
+      field :some_field, String, null: true
+    end
+
+    class Subscription < GraphQL::Schema::Object
+      field :some_field, String, null: true
+    end
```

Comment:
Not sure if this is all necessary. Without defining and passing these in below, I got an error:
`GraphQL::Schema::InvalidTypeError: Unexpected traversal member: Mutation (Class)`

Reply:
Weird, in any case, seems like a reasonable change.